pyodide_build/pywasmcross.py

```diff
@@ -59,10 +59,37 @@ def collect_args(basename):
         path = path.replace(str(ROOTDIR) + ':', '')
     env['PATH'] = path
 
+    skip_host = os.environ.get('SKIP_HOST', 'True').lower() == 'true'
```

Comment:
Not sure about the canonical way to pass boolean parameters between Python processes using environement variables (that need to be str). This works but is a bit awkward.

Reply:
I think the typical UNIX thing for boolean environment variables is just that existing at all is considered True.  So maybe for the sending do:

```
if pkg.get('build', {}).get('skip_host', True)):
  env['SKIP_HOST'] = ''
```

and receiving do:

```
'SKIP_HOST' in os.environ
```

Reply:
Yes, that would work better -- updated it.